includes/fs-core-functions.php

```diff
@@ -66,13 +66,25 @@ function fs_get_template( $path, &$params = null ) {
 	--------------------------------------------------------------------------------------------*/
 	function fs_enqueue_local_style( $handle, $path, $deps = array(), $ver = false, $media = 'all' ) {
 		global $fs_core_logger;
+
+		$css_path    = fs_normalize_path( WP_FS__DIR_CSS );
```

Comment:
@fajardoleo the following 4 lines (including the current one), are repeating in `fs_img_url()` but with `WP_FS__DIR_IMG` instead. You can create a global helper function for code reusability.

And BTW, isn't a similar logic should be added to `fs_enqueue_local_script()`?